schemas/gateway-schema.json

```diff
@@ -22,56 +22,388 @@
                                         },
                                         "debug": {
                                             "type": "boolean",
-                                            "description": "Enable debug logs in gateway service"
+                                            "description": "Enable debug logs in gateway service",
+                                            "default": false
+                                        },
+                                        "sslDebug": {
+                                            "type": "string",
+                                            "description": "Configure attributes fro SSL config, see java property javax.net.debug."
                                         },
                                         "heap": {
                                             "type": "object",
                                             "description": "Configure JVM parameters for heap size",
                                             "properties": {
                                                 "init": {
                                                     "type": "integer",
-                                                    "description": "Initial heap size in MB."
+                                                    "description": "Initial heap size in MB.",
+                                                    "default": 32
                                                 },
                                                 "max": {
                                                     "type": "integer",
-                                                    "description": "Maximum heap size in MB."
+                                                    "description": "Maximum heap size in MB.",
+                                                    "max": 512
+                                                }
+                                            }
+                                        },
+                                        "apimlId": {
+                                            "type": "string",
+                                            "pattern": "^[a-z0-9]{1,64}$",
```

Comment:
is it a requirement to be lowercase?

Reply:
I've fixed it to support both lower and upper

Reply:
I guess it should be as the serviceId, only lowercase